### PR TITLE
feat: add orm client to typescript webhooks

### PR DESCRIPTION
**Pull request by leoalves**
## Motivation and Context

Currently Typescript Webhooks do not expose the new ORM.  This pulls request adds the ORM to the Webhook context

#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by jensneuse — APPROVED** [review]
I think it looks good overall. We might want to add a test to the teatsuite package that uses the ORM within a webhook to fixate this feature.

**Comment by leoalves** [comment]
> I think it looks good overall. We might want to add a test to the teatsuite package that uses the ORM within a webhook to fixate this feature.

I can try to do that. Just need to figure it out how the test suite works.  I probably won't have time this weekend but should be able to do it Monday.
Thanks

**Comment by leoalves** [comment]
I added a new project to the testsuite package (`/packages/testsuite/apps`) to test the addition of the ORM to the webhook.  But when the webhook calls the GraphQL server it returns 404. My guess is that the test server doesn't have the GraphQL server.  The test just throws an error that it's not able to form the JSON. This might be a bug in the ORM package, it's not validating if the response has the `.data` key or not.  

Is there another way to test it the ORM in the webhook?  I saw that there is another folder (`/testapps/default`).  In that project webhook I added a call to the ORM.  But I am not sure how to test it.  I didn't see anywhere where those operations are called from a test file.   

Thanks

**Review by jensneuse — APPROVED** [review]

**Review by Aenimus — APPROVED** [review]

**Comment by jensneuse** [comment]
@leoalves thank you for contributing this! We'll make a release soon.

File: pkg/apihandler/internalapihandler.go [hunk 1]
```diff
@@ -124,38 +121,18 @@ func (i *InternalBuilder) BuildAndMountInternalApiHandler(ctx context.Context, r
 		}
 	}
 
-	if api.EnableGraphqlEndpoint {
```
Comment (leoalves) [inline pkg/apihandler/internalapihandler.go:127]: Does this mean the `/graphql` endpoint will be public now?

Comment (jensneuse) [reply pkg/apihandler/internalapihandler.go:127]: No, it's internal.

**Pull request merged**